### PR TITLE
various big number fixes

### PR DESCRIPTION
**Pull request by gau94rav**
New big number was developed in rush with many ideas that make it too complex, this PR aims to clean it a bit and also fix several issues.

It also solves this [jira](https://snyksec.atlassian.net/jira/software/c/projects/DP/boards/443?modal=detail&selectedIssue=DP-325&assignee=62261dde94f7e200690091ae) ticket as well.

The tooltip part for big number in this PR depends on this [PR](https://github.com/topcoat-data/topcoat-public/pull/121).

### CODE REVIEW

**Comment by sareg0** [comment]
Are there any core changes we should be aware of? 
Can you give some testing insctructions?

**Comment by gau94rav** [comment]
> Are there any core changes we should be aware of? Can you give some testing instructions?



> Are there any core changes we should be aware of? Can you give some testing instructions?

Hi @sareg0 the easiest way to test this is by switching to this branch and using this syntax:

`<t-big-number :number-value="29" :number-previous="40" label="Big Num" />`

This component can easily be used without any layer.

But if you want to use with a layer you can use this simple query for test.

`SELECT 28 as value, 40 as previous`

File: library/components/TTooltip.vue [hunk 1]
```diff
@@ -38,14 +38,19 @@ export default {
 			type: String,
 			default: "top"
 		},
+		isVisible: { // To control state from a component.
+			type: Boolean,
+			default: false,
+		}
```
Comment (gau94rav) [inline library/components/TTooltip.vue:44]: @sareg0 You were right about controlling it from component, but at the same time we need internal state as well for pages 😅, I am thinking about having both, I have seen this approach in other CLs as well, let me know if you have thoughts on it.

**Comment by gau94rav** [comment]
We need a linting tool for public repo,  the code is always badly linted after pushed.

File: library/components/TTooltip.vue (hunk 1 above)
Comment (sareg0) [reply library/components/TTooltip.vue:44]: suggestion: split the big number and tooltip fixes

File: visualizations/frontend/other/TBigNumber.vue [hunk 2]
```diff
@@ -1,174 +1,212 @@
...
+					<slot name="tooltip"></slot>
+				</t-tooltip>
+			</div>
+		</div>
+
+		<div class="mt-3">
+			<span class="font-normal leading-8" :class="fontSizes[bigNumberSize]">
+				<div class="flex items-center gap-2">
+					<t-tooltip position="right">
+						<span slot="trigger" :style="{ color: numberTextColor }">
```
Comment (gau94rav) [inline visualizations/frontend/other/TBigNumber.vue:27]: This was a old already merged code of Meg, re-added it to new syntax inside tooltip.

File: library/components/TTooltip.vue (hunk 1 above)
Comment (gau94rav) [reply library/components/TTooltip.vue]: hi @sareg0 splitted the tooltip [PR](https://github.com/topcoat-data/topcoat-public/pull/134)

**Review by sareg0 — APPROVED** [review]
Unfortunately I can't really see what's changing very well. I trust your judgement.

**Pull request merged**